Extensions/Effects/reflection-pixi-filter.ts

```diff
@@ -1,78 +1,124 @@
-// @ts-nocheck - TODO: fix typings in this file
-
 namespace gdjs {
+  interface ReflectionFilterExtra {
+    _animationTimer: number;
+    animationSpeed: number;
+    animationFrequency: number;
+  }
   gdjs.PixiFiltersTools.registerFilterCreator(
     'Reflection',
     new (class extends gdjs.PixiFiltersTools.PixiFilterCreator {
       makePIXIFilter(layer, effectData) {
         let time = 0;
-        const reflectionFilter = new PIXI.filters.ReflectionFilter(
-          effectData.booleanParameters.mirror,
-          effectData.doubleParameters.boundary,
-          [
+        const reflectionFilter = new PIXI.filters.ReflectionFilter({
```

Comment:
The constructor also accepts an object?

Reply:
The type definition only accept an object. We probably used the legacy way.